src/pages/LessonPage.scss

```diff
@@ -3,10 +3,8 @@
 
 .container {
   max-width: 800px;
-  padding-left: 15px;
```

Comment:
I stedet for å rote med margins her og i Footer.scss, synes jeg vi heller skal sette `background-color` til black i container til App.js / App.scss.